documentation/views/control.rst

```diff
@@ -6,57 +6,15 @@ Flexibility opportunities
 
 Flexibility opportunities have commercial value that users can valorise on.
 When FlexMeasures has identified commercial value for flexibility opportunities, the user is suggested to act on them.
```

Comment:
commercial value for flexibility opportunities -> commercial value for flexibility
on them -> on it